internal/services/managedredis/managed_redis_data_source.go

```diff
@@ -255,8 +274,8 @@ func (r ManagedRedisDataSource) Read() sdk.ResourceFunc {
 						ClusteringPolicy:                pointer.FromEnum(props.ClusteringPolicy),
 						EvictionPolicy:                  pointer.FromEnum(props.EvictionPolicy),
 						GeoReplicationGroupName:         flattenGeoReplicationGroupName(props.GeoReplication),
-						GeoReplicationLinkedDatabaseIds: flattenLinkedDatabases(props.GeoReplication.LinkedDatabases),
 						Module:                          flattenModules(props.Modules),
+						Persistence:                     flattenPersistence(props.Persistence),
```

Comment:
I rememeber I was commented the data source shall use its own flatten functions instead of sharing the same one with resource. 
But considering this data source is already sharing these functions, it may be acceptable. 
IMO we can keep leave this question to maintainers and if this is the case, the "best practices" document shall be updated